examples/js/postprocessing/EffectComposer.js

```diff
@@ -192,6 +192,20 @@ THREE.Pass = function () {
 
 };
 
+// Helper for passes that need a scene that simply has the whole viewport filled with a single quad.
+THREE.Pass.createFillQuadScene = function( material ) {
+
+	var fillQuad = {};
+
+	fillQuad.camera = new THREE.OrthographicCamera( - 1, 1, 1, - 1, 0, 1 );
+
+	fillQuad.quad = new THREE.Mesh( new THREE.PlaneBufferGeometry( 2, 2 ), material );
+	fillQuad.quad.frustumCulled = false; // Avoid getting clipped
```

Comment:
I think this is not needed either?

Reply:
Probably not, I just thought there was some reason to have it since so many of the postprocessing passes used it.